### ISSUE TITLE
develop branch build fails

### ISSUE THREAD

**Issue by yoonlees** [cites-illinois/elmr#1]
Build is failing due to the unit test fails. 

Current commit that fails: 4fc3b61638a566c04ae2d940223254e71bc0154d
Last successful build: 628a49253f08766b0bd8acd40148005ffbaca2dd

**Comment by argherna** [comment]
Specific error:

```
[ERROR] Tests run: 6, Failures: 2, Errors: 0, Skipped: 0, Time elapsed: 0.147 s <<< FAILURE! - in edu.illinois.techservices.elmr.servlets.SessionServletTest
[ERROR] testCreateSessionNoRequestParameter  Time elapsed: 0.084 s  <<< FAILURE!
org.opentest4j.AssertionFailedError: expected: <302> but was: <503>
	at edu.illinois.techservices.elmr.servlets.SessionServletTest.testCreateSessionNoRequestParameter(SessionServletTest.java:100)
```

Oops. This one slipped out. I'll get it fixed. Sorry.

**Comment by argherna** [comment]
This error occurs because of 96c36a75e90e0c66551f0cf2ae8ef78b7d32f674. When the fallback implementation was removed, a redis instance (or other external datastore) was **required** for the tests to work. This explains why a 503 status is returned in the test (this is the correct behavior when there is no external store).

To remedy this, I'm going to keep the test, but make sure it runs only when the system property `edu.illinois.techservices.elmr.redis.CanConnect` is set on the build. This is already documented and already is used for this purpose in other tests.

**Comment by argherna** [comment]
Fix commited 1dc3f73f49e91a449c334539a26c7a9709f124c8.

**Comment by ddriddle** [comment]
Unit tests now pass in Dockerfile. Thanks!
